C/primitive/elements/primitiveJetNode.inc

```diff
@@ -461,6 +485,18 @@
 , .sourceIx = ty_u
 , .targetIx = ty_mspbw256w64
 }
+,[CURRENT_ISSUANCE_ASSET_PROOF] =
+{ .tag = JET
+, .jet = current_issuance_asset_proof
+, .sourceIx = ty_w32
```

Comment:
These should also be changed to be ones rather than w32s. (This and the entry immediately following it.)

Reply:
Fixed.